Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Execution_of_an_uncommon_process_at_an_early_startup_stage.yml

```diff
@@ -0,0 +1,1783 @@
+    scriptarguments:
+      command_line:
+        complex:
+          root: alert
+          accessor: targetprocesscmd
+          transformers:
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: alert.osparentcmd
+                iscontext: true
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: alert.cgocmd
+                iscontext: true
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1141,
+          "y": 804
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "50":
+    id: "50"
+    taskid: b2fc3097-f184-4bb8-840d-bb47794cac9d
+    type: title
+    task:
+      id: b2fc3097-f184-4bb8-840d-bb47794cac9d
+      version: -1
+      name: Global Prevalence Check
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "47"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1435,
+          "y": 1067
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "52":
+    id: "52"
+    taskid: 2761b831-f9ec-4442-80f6-b8ba9cb19427
+    type: condition
+    task:
+      id: 2761b831-f9ec-4442-80f6-b8ba9cb19427
+      version: -1
+      name: Is the action process globally prevalent?
+      description: This task evaluates the prevalence check of the Action process SHA256. If the file's SHA256 is globally prevalent, the alert is closed as a false positive; otherwise, the task proceeds to the next task.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "56"
+      Global prevalence:
+      - "5"
+    separatecontext: false
+    conditions:
+    - label: Global prevalence
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.AnalyticsPrevalence.Hash.value
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1435,
+          "y": 1319
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "53":
+    id: "53"
+    taskid: c099bc6b-52da-49e2-8e33-f13ac5dec004
+    type: regular
+    task:
+      id: c099bc6b-52da-49e2-8e33-f13ac5dec004
+      version: -1
+      name: Get alert extra data
```

Comment:
Please give a meaningful name. Based on the fields you use from the returned data.

Reply:
I use this command to retrieve multiple fields, so I believe it's best to keep the name as is.
Here are the fields I use from the core extra data fields:
Core.OriginalAlert.event.action_process_image_path
Core.OriginalAlert.event.event_time_delta_from_agent_host_boot_time_seconds
Core.OriginalAlert.event.is_action_process_elevated
Core.OriginalAlert.event.is_action_process_image_created_recently
Core.OriginalAlert.event.is_vendor_highly_rare
Core.OriginalAlert.event.is_low_entropy
Core.OriginalAlert.event.is_action_process_image_extension_non_exe
Core.OriginalAlert.event.is_action_process_path_not_program_files
Core.OriginalAlert.event.is_high_entropy
Core.OriginalAlert.event.is_action_signature_invalid

Reply:
ACK.